app/client/packages/rts/src/ctl/move-to-postgres.mjs

```
@@ -192,20 +106,15 @@ function transformFields(obj) {
     } else if (key === "_class") {
       const type = mapClassToType(obj._class);
       if (type) {
-        obj.type = type; // Add the type field
+        obj.type = type;
       }
-      delete obj._class; // Remove the _class field
+      delete obj._class;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)